### PR TITLE
Click on clear should not close keyboard

### PR DESCRIPTION
**Pull request by hakubo**
- focus on input after clear search is clicked    
- click area have 30px on side-nav clear now

### CODE REVIEW

**Comment by vforge** [comment]
LGTM

**Comment by bognix** [comment]
LGTM

File: front/main/app/styles/component/_side-nav.scss [hunk 1]
```diff
@@ -159,6 +138,27 @@
 	}
 }
 
+.side-search__input,
+.side-search__input:focus {
+	background-color: $color-white;
+	border: 1px solid $search-box-border-color;
+	border-radius: $border-radius-global;
+	height: rem-calc(28);
+	margin: rem-calc(8) rem-calc(13);
```
Comment (bkoval) [inline front/main/app/styles/component/_side-nav.scss]: Just a side question - why do you use rems for defining the box model? I was pretty sure we had a signal from the UX team that we should be using pixels for defining paddings / margins.

Comment (hakubo) [inline front/main/app/styles/component/_side-nav.scss]: this signal did not propagate here yet. It sounds reasonable, but I won't be changing it now.

**Pull request merged**